### PR TITLE
adding nteract v0.0.14

### PR DESCRIPTION
**Pull request by laurentperrinet**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

**Comment by miccal** [comment]
Can you please add an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

File: Casks/nteract.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'nteract' do
+  version '0.0.14'
+  sha256 '0734cc687cdf64f499c760e951ea27810d8ddd05d5708f9384fcaeea82117108'
+
+  url "https://github.com/nteract/nteract/releases/download/v0.0.14/nteract-#{version}.dmg"
```
Comment (miccal) [inline Casks/nteract.rb]: `url "https://github.com/nteract/nteract/releases/download/v#{version}/nteract-#{version}.dmg"`

**Comment by meduz** [comment]
I have not seen any trace of an appcast on https://github.com/nteract/nteract ...

**Comment by vitorgalvao** [comment]
@meduz The document @miccal linked you to explains:

> * If the app is distributed via GitHub releases, the `appcast` will be of the form `https://github.com/{{user}}/{{project}}/releases.atom`. (Example Cask: [`electron`](https://github.com/caskroom/homebrew-cask/blob/161f85b605e160ff96e7dd11732d85609e13dc51/Casks/electron.rb#L6L7))

So `https://github.com/nteract/nteract/releases.atom`.

File: Casks/nteract.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'nteract' do
+  version '0.0.14'
+  sha256 '0734cc687cdf64f499c760e951ea27810d8ddd05d5708f9384fcaeea82117108'
+
+  url "https://github.com/nteract/nteract/releases/download/v#{version}/nteract-#{version}.dmg"
+
+  name 'nteract'
+  homepage 'https://github.com/nteract'
```
Comment (vitorgalvao) [inline Casks/nteract.rb]: `https://github.com/nteract/nteract`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Stanzas are all over the place. You need to rerun the `style --fix` command.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**